build/build_iso

```diff
@@ -170,7 +170,7 @@ if [ -n "$REFRESH_KAYAK" ]; then
 		echo "REFRESH $f"
 		cp `echo $f | sed 's^.*/kayak/^^'` $f
 	done
-	rsync -avr $PROTO/boot/ $ISO_ROOT/boot/
```

Comment:
no more copy ?

Reply:
That's just something that is useful when testing loader changes - it only does/did anything if REFRESH_KAYAK is set in the environment.